README.md

```diff
@@ -19,7 +19,7 @@ This is a CLI tool for generating OTP codes from a backup vault from the Android
 
 ### Installation
 
-The easiest way to install Aegis-rs is by using `cargo`:
+The easiest way to install Aegis-rs is by using [cargo](https://github.com/rust-lang/crates.io/):
```

Comment:
```suggestion
The easiest way to install Aegis-rs is by using [cargo](https://crates.io/):
```

Reply:
That's what I thought too, but then thought maybe I should use a GitHub link.
So please commit your edit! Thanks.